packages/ember-glimmer/lib/components/curly-component.js

```diff
@@ -1,6 +1,7 @@
 import { StatementSyntax, ValueReference } from 'glimmer-runtime';
 import { AttributeBindingReference, RootReference, applyClassNameBinding } from '../utils/references';
 import EmptyObject from 'ember-metal/empty_object';
+import Dict from 'ember-metal/dictionary';
```

Comment:
The object is not exposed to users either, so it's probably not very important for it to be an empty object either